### PR TITLE
Replaced graphiql with GraphQL playground

### PR DESCRIPTION
**Pull request by rubanooo**
GraphQL Playground provides a much nicer developer experience with multiple tabs and clearer documentation.

* I have used the official template from the [official repository](https://github.com/graphql/graphql-playground#as-html-page ).
* I added all static assets as local files and removed the old ones

### CODE REVIEW

**Comment by weisswurstkanone** [comment]
@rubanooo , looks great but one question:

The other two changes are not directly related to this improvement, right? Maybe we could split it up in to two separate PRs ?

* Fixed the token header in the controller action
* Added the API token as header instead of using the query parameters

**Comment by rubanooo** [comment]
@weisswurstkanone Sure, do you want me to take those two changes out of this PR and create a new PR to pimcore:1.0 with those changes?

**Comment by weisswurstkanone** [comment]
yes please if it is easily possible. thx!

**Comment by weisswurstkanone** [comment]
@rubanooo 🏓

**Comment by rubanooo** [comment]
Sorry for the delay, I was on holiday 😃. I have created a PR for the first item. I don't think we need a PR for _Added the API token as header instead of using the query parameters_ because the query parameter is still a valid way to pass the token.

File: src/Command/GraphQL/RebuildDefinitionsCommand.php [hunk 1]
```diff
@@ -33,10 +33,10 @@ protected function configure()
             ->setName('datahub:graphql:rebuild-definitions')
             ->setDescription('Rebuild GraphQL endpoint definitions')
             ->addOption(
-            'definitions',
+                'definitions',
                 null,
                 InputOption::VALUE_REQUIRED,
-            'Comma separated list of endpoints'
+                'Comma separated list of endpoints'
```
Comment (weisswurstkanone) [inline src/Command/GraphQL/RebuildDefinitionsCommand.php:39]: why ?

Comment (rubanooo) [reply src/Command/GraphQL/RebuildDefinitionsCommand.php:39]: I might have merged 1.0 in my branch, these are not my changes :(

**Comment by weisswurstkanone** [comment]
Hello @rubanooo This looks really fancy now 👍 Could you please sign the CLA ? Thanks a lot!

**Comment by rubanooo** [comment]
> Hello @rubanooo This looks really fancy now 👍 Could you please sign the CLA ? Thanks a lot!

I have, but another contributor who has commited to the 1.0 branch has not.

**Comment by weisswurstkanone** [comment]
Merci!

**Pull request merged**